.claude/skills/autoresearch/SKILL.md

```
@@ -0,0 +1,272 @@
+|--------|---------|
+| "Should be done now" | Run the quality gate |
+| "I'm confident in these findings" | Confidence != Evidence |
+| "The search results confirmed it" | Show the URL and cited text |
+| "I covered the main points" | Check question_answer_rate >= 0.9 |
+| "Further research would be diminishing returns" | Only the quality gate decides that |
+
+## Invocation & Bootstrap
+
+### Argument Parsing
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)